src/rdfaparser.js

```diff
@@ -179,7 +179,7 @@ class RDFaProcessor {
 
   static parseRDFaDOM (dom, kb, base) {
     var p = new RDFaProcessor(kb, { 'base': base })
-    dom.baseURI = base
+    // dom.baseURI = base  // @@ Not allowed 2017- readonly property -- necessary?
```

Comment:
Why readonly? That's required as far as I can see from the CLI. It passes the baseURI, then in 'process' (line 266), it brings it back to node. A bit awkward that it combines the document node with other info but it is convenient. I suppose line 731 is required as well.